### PR TITLE
README includes Documentation and install instructions

### PR DESCRIPTION
**Pull request by t-ekla**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,19 +1,41 @@
...
 
-A few motivating and useful examples of how your product can be used. Spice this up with code blocks and potentially more screenshots.
+## Testing
 
-## Development setup
+This programme uses PyTest.
 
-Describe how to install all development dependencies and how to run an automated test-suite of some kind. Potentially do this for multiple platforms.
+```
+> python3 -m pytest
```
Comment (breycarr) [inline README.md:25]: if they want coverage, they can run pytest --cov

File: README.md [hunk 2]
```diff
@@ -1,19 +1,41 @@
...
+```
+
+## Documentation
+
+We have a Project C A C T U S [blog](https://medium.com/@makers_c_a_c_t_u_s) on Medium, documenting significant moments, such as setting up the hardware for the first time or a fixing a particularly stubborn error.
+
+We used [Trello](https://trello.com/b/DZAhiebz/c-a-c-t-u-s) to divide our daily tasks and assign tickets to pairs. Checkout this repo's [wiki](https://github.com/breycarr/deep_root/wiki) for our group's daily diary using agile methodologies. You can also find the project's:
+- [MVP](https://github.com/breycarr/deep_root/wiki/MVP)
+- [Cacti Garden](https://github.com/breycarr/deep_root/wiki/Cacti-Garden)
```
Comment (breycarr) [inline README.md:34]: Very important addition!

File: README.md [hunk 3]
```diff
@@ -1,19 +1,41 @@
...
-Describe how to install all development dependencies and how to run an automated test-suite of some kind. Potentially do this for multiple platforms.
+```
+> python3 -m pytest
+```
+
+## Documentation
+
+We have a Project C A C T U S [blog](https://medium.com/@makers_c_a_c_t_u_s) on Medium, documenting significant moments, such as setting up the hardware for the first time or a fixing a particularly stubborn error.
+
+We used [Trello](https://trello.com/b/DZAhiebz/c-a-c-t-u-s) to divide our daily tasks and assign tickets to pairs. Checkout this repo's [wiki](https://github.com/breycarr/deep_root/wiki) for our group's daily diary using agile methodologies. You can also find the project's:
```
Comment (breycarr) [inline README.md:32]: Is our trello public?

**Review by breycarr — APPROVED** [review]
Thanks Frankie, looks really good!

File: README.md (hunk 3 above)
Comment (t-ekla) [reply README.md:32]: Will check now and set it if not, good spot

File: README.md (hunk 1 above)
Comment (t-ekla) [reply README.md:25]: Thanks, will add that to next update

**Pull request merged**